image/assets/lang/English.json

```diff
@@ -495,12 +495,12 @@
   "app_phonebook_contact_title": "Add contact",
   "app_phonebook_contact_no_name": "no name",
   "app_phonebook_contact_information": "Information",
-  "app_phonebook_contact_flag_fav": "FAVOURITES",
+  "app_phonebook_contact_flag_fav": "FAVORITES",
   "app_phonebook_contact_flag_speed_dial": "SPEED DIAL",
   "app_phonebook_contact_flag_ice": "ICE",
   "app_phonebook_contact_flag_blocked": "BLOCKED",
   "app_phonebook_ice_contacts_title": "Emergency Contacts",
-  "app_phonebook_favourite_contacts_title": "Favourite Contacts",
+  "app_phonebook_favorite_contacts_title": "Favorites",
```

Comment:
you know that both forms are correct? :P

Reply:
One is British English (favourites), and the other is American English (favorites).
We use American English (at least in designs).